kimera_pgmo/app/offline_deformation.cpp

```diff
@@ -0,0 +1,91 @@
+#include <config_utilities/config.h>
+#include <config_utilities/parsing/yaml.h>
+#include <config_utilities/printing.h>
+#include <config_utilities/validation.h>
+#include <config_utilities/virtual_config.h>
+#include <gflags/gflags.h>
+#include <glog/logging.h>
+#include <kimera_pgmo/deformation_graph.h>
+#include <kimera_pgmo/optimizer/kimera_rpgo_optimizer.h>
+
+#include <filesystem>
+DEFINE_string(dgrf, "", "dgrf file to read");
+DEFINE_string(config, "", "path to config to use");
+DEFINE_string(log_path, "", "dgrf file to read");
+
+namespace kimera_pgmo {
+
+struct OfflineDeformationConfig {
+  std::string dgrf_file;
+  std::string log_path;
+  // optimizer
+  config::VirtualConfig<Optimizer> optimizer{KimeraRpgoOptimizer::Config()};
+};
+
+void declare_config(OfflineDeformationConfig& config) {
+  using namespace config;
+  name("OfflineDeformationConfig");
+  field(config.dgrf_file, "dgrf_file");
+  field(config.log_path, "log_path");
+  field(config.optimizer, "optimizer");
+}
```

Comment:
(minor) You could also make use of the command line parsing in `config_utilities` instead of using gflags. I'd probably write the CLI to be something like:
```
./offline_deformation path/to/deformation [path/to/output] --config-utilities-file path/to/optimizer_config
```
and parse the input path and deformation graph separately as positional args (alternatively you could keep them as config fields and do `--config-utilities-yaml {log_path: /path/to/log, dgrf_file: /path/to/dgraph}` and get the path normalization / expansion with `field<Path::Absolute>` but that seems kinda annoying)

Feels like a next time kind of thing though